easier68k/core/opcodes/bcc.py

```diff
@@ -0,0 +1,737 @@
+# -*- coding: utf-8 -*-
+
+from ...core.enum.condition_status_code import ConditionStatusCode
+from ...simulator.m68k import M68K
+from ...core.opcodes.opcode import Opcode
+from ...core.util import opcode_util
+from ..util.parsing import *
+
+# makes TC way easier lol
+from bitstring import Bits
+from math import log
+COND_CODE_TO_OPCODE = {
+    '\x00': 'BRA',
+    '\x02': 'BHI',
+    '\x03': 'BLS',
+    '\x04': 'BCC',
+    '\x05': 'BCS',
+    '\x06': 'BNE',
+    '\x07': 'BEQ',
+    '\x08': 'BVC',
+    '\x09': 'BVS',
+    '\x0A': 'BPL',
+    '\x0B': 'BMI',
+    '\x0C': 'BGE',
+    '\x0D': 'BLT',
+    '\x0E': 'BGT',
+    '\x0F': 'BLE'
+}
+
+class branch_code(Opcode):
+    """
+    All Branch operations will inherit from branch_code.
+    there will be a conditional function to be overidden across child classes
+    one must also redefine self.cond_code across child classes
+    Additionally, a redefinition of command_matches(), from_str(), and disassemble_instruction() will suffice
+    """
```

Comment:
I think we should actually merge these two docstrings, since I think that our docs (which are WIP) would just ignore the second docstring block.